src/biotite/structure/transform.py

```diff
@@ -234,6 +234,78 @@ def rotate_about_axis(atoms, axis, angle, support=None):
     return _put_back(atoms, positions)
 
 
+def orient_principal_components(atoms):
+    """
+    Translate and rotate the atoms to be centered at the origin with the
+    principal axes aligned to x, y, and z.
+
```

Comment:
I think a short note about the meaning of the result could be helpful for users that do not know about principal component analysis.

```suggestion
    
    The resulting coordinates have the highest variance in the x-axis and the lowest variance on the z-axis.
```